desktop_version/lang/en/strings.xml

```diff
@@ -429,6 +429,7 @@
     <string english="- Press {button} to advance text -" translation="" explanation="to dismiss a textbox. Expect `ACTION`" max="40"/>
     <string english="Press ACTION to continue" translation="" explanation="***OUTDATED***" max="34"/>
     <string english="Press {button} to continue" translation="" explanation="Expect `ACTION`" max="34"/>
+    <string english="[Press TAB to toggle movement]" translation="" explanation="Tab toggles whether the game is paused or not" max="40"/>
```

Comment:
This is displayed at x=5, so the limit is 39 instead of 40 (that's probably my fault - I see `[Press {button} to return to editor]` is also set to 40 when it shouldn't be!)